docs/design.md

```diff
@@ -411,42 +415,53 @@ is choosing a timestamp guaranteed to be greater than the latest
 timestamp of any committed transaction (in absolute time). No system can
 claim consistency and fail to read already-committed data.
 
-Accomplishing this for transactions (or just single operations)
-accessing a single node is easy. The transaction supplies 0 for
-timestamp, indicating that the node should use its current time (time
-for a node is kept using a hybrid clock which combines wall time and a
-logical time). This guarantees data already committed to that node have
-earlier timestamps.
+Time for a node is maintained by a hybrid logical clock (HLC).
```

Comment:
Could you pull out the hybrid logical clock into its own chapter? It's definitely not subordinate to transactions and seems muddled up with other unrelated things where it sits now. Probably you want it to precede the transaction chapter, keep it basic and then go more into the actual timestamp handling wrt transactions here.